test/PowerOfThreeTest.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@RunWith(Parameterized.class)
+public class PowerOfThreeTest {
+    int number;
+    boolean expectedResult;
+    String description;
+
+    public PowerOfThreeTest(int number, boolean expectedResult,
+                        String description) {
+        this.number = number;
+        this.expectedResult = expectedResult;
+        this.description = description;
+    }
+
+    @Parameterized.Parameters
+    public static Collection numsAndTargets() {
```

Comment:
Use JUnit5, which has better way of writing parameterised tests. 
Have a look here: 
https://stackoverflow.com/questions/61483452/parameterized-test-with-two-arguments-in-junit-5-jupiter